### PR TITLE
fix: use blockTypeName instead of blockName as identifier in run tab

### PR DESCRIPTION
**Pull request by fontanierh**
Block name is not necessarily unique, only (blockType, blockName) is guaranteed to be

### CODE REVIEW

File: front/pages/[user]/a/[sId]/execute/index.jsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ function ExecuteOutputLine({
             <ChevronDownIcon className="text-gray-400 h-4 w-4" />
           )}{" "}
           <div className="inline">
-            <span class="rounded-md px-1 py-0.5 bg-gray-200 font-medium text-sm">
+            <span className="rounded-md px-1 py-0.5 bg-gray-200 font-medium text-sm">
```
Comment (fontanierh) [inline front/pages/[user]/a/[sId]/execute/index.jsx:97]: @spolu  unrelated change but this was showing an error in the console so I assumed it was a typo

Comment (spolu) [reply front/pages/[user]/a/[sId]/execute/index.jsx:97]: definitely is :+1:

**Comment by spolu** [comment]
Didn't get the time to review thoroughly enough yet. But I recall map reduce names were transformed into "BLOCK_NAME"[map] nad "BLOCK_NAME"[recuce]. I don't see that logic being removed? Am I missing something?

**Comment by fontanierh** [comment]
Yes it has been removed, it was here: https://github.com/dust-tt/dust/pull/183/files#diff-e96c9ea2ae60c2601ccdc4d4a5e3728565107aa52d72be6c0baaa2e523f93d77L375

**Review by spolu — APPROVED** [review]
LGTM :+1:

**Pull request merged**